<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>infoScreen</class>
 <widget class="QMainWindow" name="infoScreen">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Infotainment</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33); color: white; font-family: &quot;Segoe UI&quot;;</string>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>801</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton { color: white; background-color: rgb(0, 85, 255); border-radius: 10px; font-size: 24px; }
QPushButton:hover { background-color: rgb(0, 0, 200); }
QPushButton:pressed { background-color: rgb(0, 0, 150); }</string>
    </property>
    <property name="text">
     <string>Check for Updates</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ParkingButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>801</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white; background-color: rgb(255, 128, 0); border-radius: 10px; font-size: 24px;</string>
    </property>
    <property name="text">
     <string>Parking Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>811</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>36</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>WazeefaTech</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdTime">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>421</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(145, 218, 0);
border-radius: 10px; font-size: 24px;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
